index.bs

```diff
@@ -3255,7 +3255,7 @@ Classes of Changes</h4>
 			Examples of changes in this class include
 			correcting non-normative examples
 			which clearly conflict with normative requirements,
-			clarifying informative use cases or other non-normative text,
+			clarifying non-normative use cases or other non-normative text,
```

Comment:
I'm not sure this part of the correction is needed, though I don't think it is wrong. That said, if we do take it, the phrase becomes a bit redundant, and we could further simplify to something like this:
```suggestion
			clarifying various other non-normative text,
```